src/components/Calculator.js

```diff
@@ -1,35 +1,58 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
```

Comment:
- [ ] Nice use of props but you forgot to validate them so please do.

Reply:
import PropTypes from 'prop-types';
Calculator.propTypes = { calculate: PropTypes.func.isRequired };

'calculate' PropType is defined but prop is never usedeslint[react/no-unused-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/no-unused-prop-types.md)

If I do not use props, then I removed them.